### PR TITLE
Kyomic patch 1 - add yuanliangliang

### PR DESCRIPTION
**Pull request by kyomic**

### CODE REVIEW

File: Untitled Diagram.drawio [hunk 1]
```diff
@@ -0,0 +1 @@
...3XwUzPxftosLo7rpgMujCXSDKNkN/++vGygSCeaKNw4GEYeOBYzkXmCNa7Vz62mh2LuJLxs65vCeMZucSwLzdORcgyojOJYApvY1jhSTABgtHci4BTOutdi59bXQ7F5jbWzl0CHTQ7UuwIVz7Rq2ADGN6DySpt3AgEUgSv7vymTpJ3ETrgSJthxVHEsnNuh4iUDKeQL/L+jhi0ZDBC+T01jih+MwTSQvkxG+u00GjDy+XQ/wwfGsxJR8cWd7qOeGAlRHNYIOdxsYf4nuFYuEwPL6TxEb/bJR9mIbe2B+BR5KoPkyYDVBfSN2R/8S1s5z8Q+QfsKTxIPnnvMz/qz2oSgBCB8DbAFyUZfpytNuq2WH9+jVTscS4qBLtz3QganfMD/M1YFlxY/iKHF+K+Yqk8zVgmX9j+IodX2r5Cqh0vgb81oYxfCWOL8V8ye+/BqyTbQpfYvkcx5cqvkIinS9ZhR8dfLn8XjVfoWy+xG81mcaXWLKow5fLHyXzxXo/6hJIzx8jWcVJHXy5/FE1X9Ljr4j+Rny5/FExX/Ljr2jAr56NwlcC+WIuvpfMF5jpJd8/Glq/R/kiji/FfMn3j4bW71G+qONLLV8K/KOh9XuG1CfECkiOL1l8RdiPYMrly9D6PcpX4PhSzJd8/2ho/R7ly9W/FPMVSh9/jAyt36N8ufqXar6k1+9jQ+v3KF+u/qWYr8iTzpeh9XuULzd/QjVfgXS+qJl8iQXeOr/9Sx1fcvlK+j84KD1/jA2t36N8ufkTqvmKpfNlaP1eLADX4cvVJyTzRUh/gNuXDpihBXwUMFegUA6Y9ApYbGgFHwXMVShUAya/BBYbWsIX69h1AHMppGzAmPI5FLGhNXwMMLEynANMHWCJdMAMLeKjgLlJOqoBk+8iE0Or+GI1vg5grkohG7BA+TSKxNAyPgqYm6ajHDDpWaRYQfS3AMzVwVQDpsBFGlrIRwFzdTDlgMnvwQyt5KOAuTqYasAi+T2YoZV8FDA3VUc5YNLngu0tadhacAhf1IVg7/D1If2NXAkS1d/V2VXoLysJU64/c0UkFfrLypHk6u9B/X1XpVahv6wURqr+2Di+Cy9VyC8rwVAuv6vwqpBf1kQe1fKLhNPJL1V+WdNsjuQ/FhvCMMj5I/oTl/z9sv7125Lz6nj3Ml3Nv/Bp1uzxPw==</diagram></mxfile>
```
Comment (ljharb) [inline Untitled Diagram.drawio]: pretty sure this shouldn't be here

File: delegates.txt [hunk 2]
```diff
@@ -72,6 +72,7 @@ Claude Pache (CPE)
 Conrad Watt (CWT)
 Conrad Watts (CWS)
 Corey Frang (CF)
+Cuili (CLI)
```
Comment (ljharb) [inline delegates.txt:85]: who's this?

**Comment by Heatherlee85527** [comment]
Get it

**Review by Ashcjs — APPROVED** [review]

File: delegates.txt [hunk 3]
```diff
@@ -425,6 +426,7 @@ Xuan Huang (HUX)
 YaJu Wang (WYJ)
 Yan Kun Zhen (YKZ)
 Yang Guo (YGO)
+Yang Liang Liang (YLL)
```
Comment (michaelficarra) [inline delegates.txt:429]: This abbreviation is already taken: https://github.com/tc39/notes/blob/211aa3f9e0fac861bba427776a9ecb8663fdf280/delegates.txt#L256.

File: delegates.txt (hunk 2 above)
Comment (ctcpip) [reply delegates.txt:85]: https://github.com/tc39/Admin-and-Business/issues/133

Comment (ljharb) [reply delegates.txt:85]: ah, gotcha. what's their full name?

**Comment by ctcpip** [comment]
closing.  see https://github.com/tc39/Admin-and-Business/issues/133

**Pull request closed without merging**